server/server.js

```diff
@@ -10,12 +10,18 @@ const loginSignupHandler = require('./controllers/loginSignup');
 const userHandler = require('./controllers/user');
 const artistHandler = require('./controllers/artist');
 const imageHandler = require('./controllers/imageUpload');
-
+const notificationHandler = require('./controllers/notification');
 const app = express();
 busboy.extend(app, {
   upload: true,
 });
 const port = process.env.PORT || 3000;
+const io = require('./sockets.js');
+
+const server = app.listen(port, function() {
```

Comment:
is asking the server to listen to the port before using middleware ok? If so can you plz share some reading material?